src/backend/dx11/src/mirror.rs

```diff
@@ -86,6 +86,27 @@ fn map_texture_type(tt: winapi::D3D_SRV_DIMENSION) -> s::TextureType {
     }
 }
 
+fn map_container(stype: &winapi::D3D11_SHADER_TYPE_DESC) -> s::ContainerType {
+    use gfx_core::shade::Dimension as Dim;
+    //TODO: use `match` when winapi allows
```

Comment:
Oh, the method I use for enums in winapi 0.2 doesn't allow you to match on it? Well, this will be fixed in winapi 0.3 for sure because I will be making enums simple integer constants and you can definitely match on integers.